src/app/components/NavBar.tsx

```diff
@@ -332,6 +332,7 @@ function NavbarItemMobile({
   setMenuOpen,
 }: NavbarItemMobileProps) {
   const active = target === usePathname();
+  const translate = useTranslations();
```

Comment:
Please change to follow our conventions:
```suggestion
  const t = useTranslations();
```